src/main/java/com/scribassu/tracto/domain/WeekType.java

```diff
@@ -0,0 +1,12 @@
+package com.scribassu.tracto.domain;
+
+public enum WeekType {
+    ODD("чис."),
+    EVEN("знам.");
+
```

Comment:
add enum for lesson, which take place both in odd and even weeks

Reply:
+